### ISSUE TITLE
Missing @mixmaxhq/semantic-release-config npm devDependencies

### ISSUE THREAD

**Issue by sberthier** [mixmaxhq/mongo-cursor-pagination#348]
It seems that https://www.npmjs.com/package/@mixmaxhq/semantic-release-config is not publish anymore, so we cannot install `devDependencies` of this modules (with `npm ci`).

I wanted to investigate about https://github.com/mixmaxhq/mongo-cursor-pagination/issues/313 because I encounter same problem.

**Comment by bradvogel** [comment]
That package is only required by the release config and not needed for local development. We shouldn't be bundling it in devDependencies.

To unblock yourself for local development, please run `npm rm @mixmaxhq/semantic-release-config` before `npm i`

**Comment by bradvogel** [comment]
This is now fixed. Package https://www.npmjs.com/package/@mixmaxhq/semantic-release-config is now public.
